WordPress/Classes/Services/Store.swift

```diff
@@ -22,53 +22,162 @@ class StoreKitTransactionObserver: NSObject, SKPaymentTransactionObserver {
 // stored property since it's a generic class.
 struct StoreKitCoordinator {
     static let instance = StoreCoordinator(store: StoreKitStore())
+    
+    static let TransactionDidFinishNotification = "StoreCoordinatorTransactionDidFinishNotification"
+    static let TransactionDidFailNotification   = "StoreCoordinatorTransactionDidFailNotification"
+    static let NotificationProductIdentifierKey = "StoreCoordinatorNotificationProductIdentifierKey"
 }
 
+typealias PendingPayment = (planID: PlanID, productID: String, siteID: Int)
```

Comment:
I'm now wondering why you went from storing `Plan` to `PlanID, ProductID`.

Actually, since a payment is for a product, do we even need to store the plan (ID)? When the backend is ready, we'll just need to pass a site ID and a SKU along with the receipt.